### DISCUSSION TITLE
PW reset

### DISCUSSION THREAD

**Discussion by krizzziz** [koush/scrypted#339] [Q&A] [answered]
In the wiki there is a procedure being explained on how to do a pw reset.

But I am already stuck on step nr.2. Ho do I set the enviroment variable? I am using a RPI with docker installation.

_1. Stop the Scrypted service.
2. Set the environment variable SCRYPTED_RESET_ALL_USERS=true.
3. Start the server.
4. Create a new user.
5. Stop the server and remove the environment variable (or it will reset again on reboot!).
6. Start the server again for normal operation._

**Comment by koush** [comment, marked as answer]
you set it in your docker compose or docker run command.

**Comment by krizzziz** [comment]
It's not working. Even when I remove the stack within portainer. And add scrypted again, it's still requesting for my password.

I can see in that I used the enviroment variable correctly but no change:

`],
        "Env": [
            "TZ=America/New_York",
            "LD_LIBRARY_PATH=/opt/vc/lib",
            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
            "NODE_VERSION=16.15.1",
            "YARN_VERSION=1.22.19",
            "SCRYPTED_DOCKER_SERVE=true",
            "SCRYPTED_CAN_RESTART=true",
            "SCRYPTED_VOLUME=/server/volume",
            "SCRYPTED_INSTALL_PATH=/server"`
